ReadExcelFile.py

```diff
@@ -19,6 +19,9 @@
 print(my_val)
 
 #Read in values of cells C1 through C5 and print
-for i in range(1,6):
-    C_Cell = sheet.cell(i,3)
-    print(C_Cell.value)
+#Define the first value to be used as the base of the for loop.
+C_Cell = sheet.cell(1,3)
+
+for i in range(1,5):
+    C_Cell.value = C_Cell.value + sheet.cell(i+1,3).value
```

Comment:
Good job summing things up. There's one major issue and one minor one.

The major issue is that in this line, you are actually changing the value of cell C1. Now, since you don't save the spreadsheet, it doesn't matter, but your next assignment will involve saving the spreadsheet with only certain changes, and this will mess you up. Try to figure out (and if you can't, I will of course help you) how to calculate the sum without changing C_Cell.value.

The minor issue is that you're assigning the variable `i` the values 1, 2, 3, 4 and then going to row `i+1`. Try to figure out how to assign the variable `i` the values `2,3,4,5` directly and not having to do the `+1` manually.

On an unrelated note, I see that you've written `sheet.cell(i+1,3).value` in line rather than in several statements. I would have done the same, but it's mostly a matter of style. Just a thought.

Finally, know that code reviews tend to be this brutal in real life too (depending, of course, on how complex the code is and what you're doing). I view this as a good thing and not an attack; this is how both code and coders get better.